File: java/problem26.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums.length==0){
            return 0;
        }
        int currentPointer=0;
        int sweepPointer;
        for(sweepPointer=0;sweepPointer<nums.length;sweepPointer++){
            if(nums[sweepPointer]!=nums[currentPointer]){
                nums[currentPointer+1]=nums[sweepPointer];
                currentPointer++;
            }
        }
        return currentPointer+1;
    }
}

File: python/problem26.py
class Solution:
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        i = 0
        try:
            while 1:
                if nums[i] == nums[i+1]:
                    del nums[i+1]
                else:
                    i+=1
                
        except:
            
            return len(nums)
